understanding/22/focus-appearance-minimum.html

```diff
@@ -214,17 +215,24 @@ <h3>Relation to non-text color contrast</h3>
            
             <p>In the example above:</p>
             <ul>
-                <li>The checkbox has sufficient contrast with its adjacent background in default, focused, checked, and checked-focused states; </li>
-                <li>The check (tick) within the checkbox contrasts with its adjacent color; </li>
-                <li>The circular outline around the checkbox that indicates focus has a sufficient <em>change</em> of contrast.</li>
+                <li>The dark square outline of the checkbox has sufficient contrast with its adjacent light background in the default state (meeting 1.4.11 Non-text Contrast).</li>
+                <li>When the checkbox receives focus in the second image, a grey ring appears around it. The ring contrasts against the light page background sufficiently (still meeting 1.4.11). The size of the ring's dark outline is larger enough to meet the <strong>Minimum area</strong> bullet of 2.4.11 Focus Appearance. The color of the pixels making up the ring's outline versus the same pixels in the unfocused state meets the 2.4.11 <strong>Change of contrast</strong> bullet. The ring's outline also contrasts against the inside shading (another more subtle indicator of focus), so both the inner and outer edges of the ring's outline meet 2.4.11's  <strong>Adjacent contrast</strong> bullet.</li>
```

Comment:
"largeR enough"
Also, with the change to the 3rd bullet now, I don't think the outer ring needs to contrast against "all" adjacent colours. Because it is separated from the component, it doesn't have to meet any adjacent colour.

Reply:
I made it "large".
 Has the 3rd bullet change been approved? I'll need to look at the final wording, but my sense is maybe there still needs to be some comment here, as they have added in that shading as part of the focus indicator. Or are we now saying that it wouldn't matter what the degree of shading here, it would still meet? (i.e., no shading, meets; shading matches the ring color -- becoming a massive ring -- still meets) It seems to me we still care about a contrast between the light shading and the edge of the control itself. So maybe this needs to be rewritten.

Reply:
The focus indicator (that meets the 1st two bullets) is sufficient, and as the indicator isn't adjacent to the component, it missed the 3rd bullet (now). We did resolve to use that text yesterday, but as it's normative the SC (with all it's changes) will go through CFC.